src/components/Tooltip/WTooltip.stories.ts

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
+import WTooltip from './WTooltip.vue'
+
+const meta: any = {
+  title: 'Example/WTooltip',
+  component: WTooltip,
+  tags: ['autodocs'],
+  argTypes: {},
+  args: {
+    bgColor: '#000',
+  },
+} satisfies Meta<typeof WTooltip>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  render: (args: any) => ({
+    components: { WTooltip },
+    setup() {
+      return { args }
+    },
+    template: `<WTooltip v-bind="args"><template #tooltip><p class="text-white-50 w-[200px]">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p></template><template #main>
```

Comment:
Can we create a wrapper with a fixed width and height to avoid hiding the tooltip?
![image](https://github.com/user-attachments/assets/a0e30cf5-d746-4747-9088-ba319175fe84)